docs/static/js/latest-webforj-version.js

```diff
@@ -0,0 +1,23 @@
+function getLatestVer() {
```

Comment:
This should cache the response for a day, the API is publicly accessible But there’s a limit (I think 60 requests per hour per IP address)